src/nuclio/common/less/general-rules.less

```diff
@@ -12,5 +12,72 @@ body {
             }
         }
     }
+
+    .table-headers {
+        display: flex;
+        font-size: 14px;
+        font-weight: bold;
+        font-style: normal;
+        font-stretch: normal;
+        letter-spacing: normal;
+        color: @dusk-three;
+        margin-bottom: 2px;
+
+        .key-header {
+            width: 34%;
+            margin-left: 16px;
+
+            &:not(.use-type) {
+                width: 36%;
+            }
+        }
+
+        .type-header {
+            width: 12%;
+        }
+
+        .value-header {
+            width: 52%;
+
+            &:not(.use-type) {
+                width: 61%;
+            }
+        }
+    }
+
+    .table-body:not(:last-child) {
+        margin-bottom: 5px;
+    }
+
+    .create-variable-button,
+    .create-header-button,
+    .create-annotation-button,
+    .create-event-header,
+    .create-label-button,
+    .create-binding-button,
+    .create-ingress-button,
+    .create-topic-button,
+    .create-broker-button,
+    .create-subscription-button {
+        display: flex;
+        align-items: center;
+        height: 36px;
+        margin-top: 7px;
+        .duskThree(0.64);
+        background-color: @white;
+        font-size: 14px;
+        font-weight: normal;
+        font-style: normal;
+        font-stretch: normal;
+        letter-spacing: normal;
+        text-align: left;
+        color: @color;
+
+        .igz-icon-add-round {
+            margin: 3px 8px 0 15px;
+            color: @orange;
+            font-size: 24px;
+        }
+    }
```

Comment:
Hmm… Why not in table.less?

Reply:
Even if it has class `table-header` this isn't related to our tables, it's only for `key-value` component. In `table.less` we have the main class `common-table`, and these styles related to real tables. When I move `key-value-input` component to `igz-controls` folder, I will move these styles to `general-rules.less` inside `igz-controls`